browser-test/src/application_filter.test.ts

```diff
@@ -0,0 +1,149 @@
+import {
+  createTestContext,
+  dropTables,
+  loginAsAdmin,
+  loginAsProgramAdmin,
+  loginAsTestUser,
+  logout,
+  seedQuestions,
+  validateScreenshot,
+} from './support'
+
+describe('normal application flow', () => {
+  const ctx = createTestContext()
+
+  beforeAll(async () => {
+    const {page} = ctx
+    await dropTables(page)
+    await seedQuestions(page)
+  })
+
+  it('all major steps', async () => {
```

Comment:
**issue (blocking):** A lot of the code in this test case was copied from an existing test and isn't necessary for exercising the filter clear button. Please reduce the scope of this test to the minimum needed to exercise the server code changes you've introduced.